stdlib/2and3/logging/handlers.pyi

```diff
@@ -19,6 +19,12 @@ if sys.version_info >= (3, 6):
 else:
     _Path = str
 
+DEFAULT_TCP_LOGGING_PORT: int = ...
+DEFAULT_UDP_LOGGING_PORT: int = ...
+DEFAULT_HTTP_LOGGING_PORT: int = ...
+DEFAULT_SOAP_LOGGING_PORT: int = ...
+SYSLOG_UDP_PORT: int = ...
+SYSLOG_TCP_PORT: int = ...
```

Comment:
Thank you for the pull request. A minor style nit: module-level constants should have no default, not even an ellipsis.